src/encode/mod.rs

```diff
@@ -25,7 +25,7 @@ pub mod writer;
 const NEWLINE: &'static str = "\r\n";
 #[allow(dead_code)]
 #[cfg(not(windows))]
-const NEWLINE: &'static str = "\n";
+const NEWLINE: &str = "\n";
```

Comment:
interesting, I didn't know that `rustc` allowed this lifetime elision, though it makes perfect sense